app/lib/ui/views/new/new_screen.dart

```diff
@@ -39,9 +41,8 @@ class NewScreen extends StatelessWidget {
               Padding(padding: EdgeInsets.all(5.0)),
               Text('Go to test page without bottom nav'),
               RaisedButton(
-                onPressed: () => Navigator.of(context, rootNavigator: true)
-                    .push(MaterialPageRoute(
-                        builder: (context) => TestScreenThree())),
+                onPressed: () =>
+                    _navigationService.navigateTo(TestScreenThree.route),
```

Comment:
You're not supposed to use nav service in UI layer. Here's what you need to do:
```
class NewScreenViewModel extends BaseViewModel{
final NavigationService _navigationService = locator<NavigationService>();
void navigateToTestScreenThree() => _navigationService.navigateTo(TestScreenThree.route);
}
```
In your view, you can simply do: `model._navigationService();`
Note: This apply to every screen that will navigate